src/graphics/draw/ClockRenderer.cpp

```diff
@@ -366,9 +365,6 @@ void drawAnalogClockFrame(OLEDDisplay *display, OLEDDisplayUiState *state, int16
 
     // hour hand radius and y coordinate
     int16_t hourHandRadius = radius * 0.35;
-    if (isHighResolution) {
```

Comment:
just wanted to check this one was a cppcheck since it seemed different to the others.

Reply:
Yes, cppcheck called out that this code was logically equivalent to just setting false.